feature-detects/es6/spread-array.js

```diff
@@ -0,0 +1,32 @@
+/*!
+{
+  "name": "ES6 Spread array",
+  "property": "spreadarray",
+  "notes": [{
+    "name": "ECMAScript 6 language specification",
+    "href": "https://tc39.es/ecma262/#sec-array-initializer"
+  },
+  {
+    "name": "Article",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax"
+  }],
+  "caniuse": "mdn-javascript_operators_spread_spread_in_arrays",
+  "authors": ["dabretin"],
+  "tags": ["es6"]
+}
+!*/
+/* DOC
+Check if browser implements ECMAScript 6 spread syntax (in array and function calls)
+WARNING: not for object literals (=> ES7) 
```

Comment:
you can add this warning to the top doc like this:
  "warnings": ["This test is not for object literals (=> ES7)"]